scripts/build-rpm.sh

```diff
@@ -48,16 +48,13 @@ build-rpm () {
         setup.cfg \
         setup.py \
         ulauncher \
-        ulauncher.desktop.in \
+        ulauncher.desktop \
         $tmpdir \
         --exclude-from=.gitignore
 
     # This is only needed because data/preferences is in .gitignore
     cp -r data/preferences $tmpdir/data/preferences
 
-    # set version to a tag name
-    sed -i "s/%VERSION%/$version/g" $tmpdir/setup.py
```

Comment:
Now the first parameter passed into this script no longer has a purpose. It is not used any where else.

Reply:
The argument is definitely not needed now. It could be loaded like this `version=$(<ulauncher/VERSION)`, but it's still used in the resulting file name and printed in the header.

Reply:
This inconsistency was already there before and outside of the scope of the PR, but fixed in 7bf0805 now anyway